### PR TITLE
[Documentation] Updated Ubuntu section of README.md to include libpng-dev

### LINKED ISSUE #1703: [Documentation] libpng-dev missing from apt-get install list in README.md for ubuntu setup
Trying to setup a new ubuntu 16.04 VM to test out glow and ran into a little snafu in the documentation.

libpng is listed in the system requirements header (and in the macOS brew header) but not in the ubuntu section. As a result, it took a bit of cmake digging to figure out why image-classifier wasn't being built at first.

### PR DESCRIPTION
**Pull request by Oewyn**
*Description*: updated README.md to include libpng-dev package to apt-get install instructions
*Testing*: None -- documentation change only
*Documentation*: README.md updated
* Fixes #1703

### CODE REVIEW

**Comment by nadavrot** [comment]
This PR reverts @rdzhabarov 's change from last night.  @Oewyn , did you intend to revert Roman's change?

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Install the required dependencies using [Homebrew](https://brew.sh/):
 
   ```bash
   brew install cmake graphviz libpng ninja protobuf wget
-  brew install --with-toolchain llvm@6
+  brew install --with-toolchain llvm
```
Comment (rdzhabarov) [inline README.md]: please do not remove this, Glow cannot run on llvm7 which is current when `brew install`.

Comment (Oewyn) [reply README.md]: My mistake, I will update the PR shortly.

**Comment by Oewyn** [comment]
PR has been updated to keep @rdzhabarov 's changes from last night.

**Comment by nadavrot** [comment]
Thanks @Oewyn @rdzhabarov

**Pull request merged**